bin/compile

```diff
@@ -84,7 +84,7 @@ fi
 boot_cmd=${BOOTBUILD_CMD:-"boot build"}
 echo "-----> Running: $boot_cmd"
 export PATH="$build/.bootbin:$PATH"
-(cd $build && $boot_cmd 2>&1 | indent)
+(cd $build && $boot_cmd 2>&1 | indent); echo ""
```

Comment:
The line that follows always ends up indented otherwise. I'm not sure why. This just looks nicer :)